projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -31,3 +31,46 @@ frontend_publish_test_image:
     - if: '$CI_COMMIT_BRANCH == "main"'
       changes:
         - projects/frontend/cicd/version.txt
+
+frontend-shared-components-build:
+  image: "versatiledatakit/vdk-cicd-base-gui:2.0.0"
+  stage: build
+  before_script:
+    - cd projects/frontend/shared-components/gui
+  script:
+    - ../cicd/build.sh
+  retry: !reference [.retry, retry_options]
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "external_pull_request_event"'
+      changes: *frontend_shared_components_locations
+  artifacts:
+    when: always
+    reports:
+      junit:
+        - projects/frontend/shared-components/gui/reports/test-results/shared/*.xml
+        - projects/frontend/shared-components/gui/reports/test-results/shared/*.xml
+    paths:
+      - projects/frontend/shared-components/gui/package-lock.json
+    expire_in: 1 week
+
+frontend-shared-components-release:
+  image: "versatiledatakit/vdk-cicd-base-gui:2.0.0"
+  stage: build
```

Comment:
There's relase stage (https://gitlab.com/vmware-analytics/versatile-data-kit/-/blob/main/.gitlab-ci.yml#L16) for releases ... ?

Reply:
Forgot to move this to release stage, it was here for testing. 😅